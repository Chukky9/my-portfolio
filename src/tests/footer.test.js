import { screen, render } from '@testing-library/react';
import Footer from '../components/footer';

test('footer renders successfully with correct year', () => {
    const { container } = render(<Footer/>)
    expect(screen.getByText(/alozie chukwuemeka/i)).toBeInTheDocument()

    const year = new Date().getFullYear()
    expect(screen.getByText(year, { exact: false })).toBeInTheDocument()
    expect(container).toMatchSnapshot()
})