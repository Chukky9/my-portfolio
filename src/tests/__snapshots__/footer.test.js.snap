// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer renders successfully with correct year 1`] = `
<div>
  <footer>
    <span>
      Created and designed by ALOZIE CHUKWUEMEKA 
      <i
        aria-hidden="true"
        class="fa fa-copyright"
      >
         
      </i>
       
      2022
    </span>
  </footer>
</div>
`;
